Sources/SwiftFusion/Core/MatrixN.swift

```diff
@@ -1,7 +1,7 @@
 public struct Matrix3: Differentiable & Equatable & KeyPathIterable {
-  var s00, s01, s02: Double
-  var s10, s11, s12: Double
-  var s20, s21, s22: Double
+  public var s00, s01, s02: Double
```

Comment:
oh yeah? Suspicious.
Suppress impulse to defend, think about alternative before replying

Reply:
changed back and added a subscript operator for access, however there seems to be a bug in compiler for `switch`-`case`, so I replaced most but one, need to discuss with @marcrasi